bin/docker-push-base

```diff
@@ -6,8 +6,5 @@ bindir=$( cd "${BASH_SOURCE[0]%/*}" && pwd )
 
 # shellcheck source=_docker.sh
 . "$bindir"/_docker.sh
-# shellcheck source=_tag.sh
```

Comment:
need to change some script in github workflow which still call `bin/docker-push-deps`
ex: https://github.com/linkerd/linkerd2/blob/main/.github/workflows/release.yml#L91

Reply:
oh nvm, you try to merge to `alpeb/buildx` I though to `main`.